aiidalab/utils.py

```diff
@@ -32,13 +35,19 @@ def update_cache():
     pass
 
 
+@ttl_cache()
```

Comment:
I think the requests call is cached already. If I remember  correctly `requests.get()` call first returns the cached value and then runs update in the background.

Reply:
But the function now also reads from a file, that would not be cached. I can remove the caching if you feel strongly about it.